assets/src/scss/main.scss

```diff
@@ -1,5 +1,5 @@
 // wp admin login css
-.login {
+.wp-core-ui {
```

Comment:
Is this the only change you made? (everything else seems to be in `build` which I assume means it's been automatically built?(